src/containers/BooksForm.js

```diff
@@ -1,32 +1,59 @@
 import Form from 'react-bootstrap/Form';
 import Button from 'react-bootstrap/Button';
+import { useDispatch } from 'react-redux';
+import { CREATE_BOOK } from '../reducers/books';
 
 const BooksForm = () => {
-  const categories = ['Action', 'Biography', 'History', 'Horror', 'Kids', 'Learning', 'Sci-Fi'];
+  const dispatch = useDispatch();
+  const categories = [
+    'Action',
+    'Biography',
+    'History',
+    'Horror',
+    'Kids',
+    'Learning',
+    'Sci-Fi',
+  ];
+  let book = { ID: null, Title: null, category: 'Action' };
+
   return (
     <Form>
       <Form.Group controlId="exampleForm.ControlInput1">
         <Form.Label>Title</Form.Label>
-        <Form.Control type="input" placeholder="Please enter title of the book" />
+        <Form.Control
+          type="input"
+          placeholder="Please enter title of the book"
+          onChange={(event) => {
+            book.Title = event.target.value;
+          }}
+        />
       </Form.Group>
       <Form.Group controlId="exampleForm.ControlSelect1">
         <Form.Label>Category</Form.Label>
-        <Form.Control as="select">
-
-          { categories.map((category) => (
-            <option key="i">
-              {' '}
-              {category}
-              {' '}
-            </option>
-
+        <Form.Control
+          as="select"
+          onChange={(event) => {
+            book.category = event.target.value;
+          }}
```

Comment:
It would be nice if you refactor your code. Implement a handleChange function like so:
``` 
handleChange({ target: { name, value } }) {
  this.setState({
    [name]: value,
  })
}

```
 and pass it as a prop to the input onChange event.